Remaining section review (#11)
ezsystems/commerce-documentation: docs/user_manual/shop_administration/cockpit.md

- Technical subsection
    - HDD usage
    - Server load
    - ERP Status
- Active shopping carts (in last 30 minutes)
- Number of customers

> - Number of products  (Note: the number of products only respects the content model data provider.

Reviewer: ```suggestion
- Number of products  (The number of products only respects the content model data provider.
```

The number of products in the eContent is displayed in **Control Center** / **Manage eContent**.)

## Data per shop

The data in the next section refers to a specific shop.
If there is more than one shop in the installation, the shop owner can choose shop and currency.

Currently available data:

- Sales - shows orders value in time
- Orders - shows number of orders

You can select a specific date range or one of predefined date ranges.

You can also compare the chosen date range with the a range of the same length in the past.
For example, if you choose last week, you can compare it with the week before.

## Products and sales

Product and sales section shows data in tables such as:

- Bestsellers (Top 20)
- Last orders (Last 365)
- Best clients (Top 20)

You can refresh this panel separately from the whole page with the icon in the top right corner on the section's title bar.

Tabs **Last orders** and **Best clients** are only visible for users who have the Siso policy / Read customers and orders Policy. See [Permissions](permissions.md) for more information.

## Search

The search section offers the following statistics:

1\. Top search terms

> The top 50 search terms were listed including how often they were searched for and how many hits (average) were found.

Reviewer: ```suggestion
Lists top 50 search terms including how often they are searched for and how many hits (average) they have.
```
